README.md

```diff
@@ -14,6 +14,22 @@ The template supports producing both Hungarian and English theses, which can be
 
 ## Compilation
 
+Compilation might be carried out through a preferred IDE (e.g. [TexStudio](https://www.texstudio.org/)), given the same commands should be executed.
+
+### Using a build system
+
+```bash
+# Generate everything automatically, in the right order.
+# (Needs GNU make to be installed!)
+make thesis
```

Comment:
This will build both the English and the Hungarian version if I see it correctly; however that rarely makes sense, as these are only sample files and only one of them is usually used.

Reply:
Understood, but in this case, which one should be the *main* document. 😕 Because right now, they appear as "siblings" in the repository. Perhaps I could make it so that the 🇬🇧 tutorial shows calling `make thesis-en` and the 🇭🇺 tutorial tells the user to call `make thesis-hu`?

Reply:
In general we can expect that a user renames the used template (e.g. from `elteikthesis_en.tex` to `mythesis.tex`) and deletes the unused one; so I suggest there should be some option for custom parametrization.